<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterProduit</class>
 <widget class="QDialog" name="AjouterProduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel {
font-weight : bold ; 

}
#groupBox{
font-weight : bold ; 
font-size : 15px ; 
}

QPushButton {
background : #FFFFFF ; 
box-shadow:  0  0 30px white ;
transition: all 0.4s ;
}

QPushButton:hover {
background : #45A4CD ; 
}
#label_3 {
color : red ; 
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter un Produit</string>
   </property>
   <widget class="QLabel" name="prix_produit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_prix">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcourir</string>
    </property>
   </widget>
   <widget class="QLabel" name="dd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
